src/components/organisms/AdvancedLending/AdvancedLending.tsx

```diff
@@ -21,6 +21,7 @@ import {
     OrderTable,
 } from 'src/components/organisms';
 import { TabSpinner, TableType } from 'src/components/pages';
+import { ThreeColumnsWithTopBar } from 'src/components/templates';
```

Comment:
I am re-applying ThreeColumnsWithTopBar in AdvancedLending page, so I updated it to handle the re-ordering of elements. Also applied it to Itayose page